### PR TITLE
complete functionality: login, signup, oAuth, search files

### PR DESCRIPTION
**Pull request by Pragati1610**
- Search files functionality: create a new file, write to a file, rename, and delete the file

### CODE REVIEW

File: public/script.js [hunk 1]
```diff
@@ -9,7 +9,13 @@ const output = document.getElementById("render");
 // Emit events
 
 btn.addEventListener("click", function(){
-    socket.emit("cmd", command.value);
+    var val = {
+        token: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvaG4uZG9lQGV4YW1wbGUuY29tIiwidXNlcklkIjoiNWY0NzQ2NjY4NzJkNmExNDFmNTNkYTIwIiwiaWF0IjoxNTk4NTA2NjMxLCJleHAiOjE1OTg1MTAyMzF9.2xlDLF3RN37BWBO172bEpNr1ydRH5DdSTGDSB4hjiLk",
```
Comment (L04DB4L4NC3R) [inline public/script.js:13]: Invalidate this token

Comment (L04DB4L4NC3R) [reply public/script.js:13]: Or change JWT secret

**Comment by L04DB4L4NC3R** [comment]
@Pragati1610 read the comments and resolve the merge conflicts

File: public/script.js (hunk 1 above)
Comment (Pragati1610) [reply public/script.js:13]: changed jwt secret

**Pull request merged**